examples/acquiring_data/goes_xrs_example.py

```diff
@@ -0,0 +1,130 @@
+example, there are times when GOES 13, 14 and 15 overlap and such data is
+available from each satellite. Similarly there are times when GOES 16 and 17 overlap.
+
+Lets query the GOES XRS data over a specified timerange:
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from sunpy import timeseries as ts
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+#############################################################
+# Lets first define our start and end times and query using the
+# `~sunpy.net.Fido`.
+tstart = "2015-06-21 01:00"
+tend = "2015-06-21 23:00"
+result = Fido.search(a.Time(tstart, tend), a.Instrument("XRS"))
+print(result)
+
+#############################################################
+# As we can see this now returns three results, one file for GOES
+# 13, one for GOES 14 and one for GOES 15, which can be identfied
+# by the `SatelliteNumber` column. However, we probably will only want
+# one of these files for our analysis, so we can query by the `attrs`:
+# `a.goes.SatelliteNumber` to specify what GOES satellite number we want
+# to use.
+result_goes15 = Fido.search(a.Time(tstart, tend), a.Instrument("XRS"), a.goes.SatelliteNumber(15))
+print(result_goes15)
+
+#############################################################
+# Now we can see that this returns just one file for the GOES 15 data.
+# Lets now download this data using `~sunpy.net.Fido.fetch`.
+file_goes15 = Fido.fetch(result_goes15)
+
+#############################################################
+# Also just to note, if this will download the file to the
+# `~/sunpy/data/` directory on your local machine. You can also
+# define where you want this to download to using the `path` keyword
+# argument in .fetch (e.g. Fido.fetch(result, path=".\")).
+
+#############################################################
+# Lets now load this data into a `~sunpy.timeseries.TimeSeries`,
+# and inspect the data using `.peek()`
+goes_15 = ts.TimeSeries(file_goes15)
+goes_15.peek()
+
+###############################################################
+# We can also pull out the individual GOES chanels and plot. The 0.5-4 angstrom
+# channel is known as the "xrsa" channel and the 1-8 angstrom channel is known
+# as the "xrsb" channel.
+fig, ax = plt.subplots()
+ax.plot(goes_15.index, goes_15.quantity("xrsb"))
+ax.set_ylabel("Flux (Wm$^{-2}$)")
+ax.set_xlabel("Time")
+fig.autofmt_xdate()
+plt.show()
+
+###############################################################
+# We can also truncate the data for the time of the large flare,
+# and analyze the different channels. For example, we can plot the
+# derivative which is useful in terms of the Neupert effect when analyzing
+# flares
+goes_flare = goes_15.truncate("2015-06-21 09:35", "2015-06-21 10:30")
+fig, ax = plt.subplots()
+ax.plot(goes_flare.index, np.gradient(goes_flare.quantity("xrsb")))
+ax.set_ylabel("Flux (Wm$^{-2}$$s^{-1}$)")
+fig.autofmt_xdate()
+plt.show()
+
+"""
```

Comment:
Do we want this to be a HTML block like the other sections?

Reply:
Yeah I think so - I was just trying to make `GOES 16 and 17 Data` a subtitle if that makes sense?

Reply:
Yup, at least to me...

Reply:
Yeah it does but I have no idea how to do that with the gallery.

Reply:
Am monitoring RTD for the output if that helps?

Reply:
BTW for the context: I think the code snippet has been made on the assumption that `markdown` or something similar can be applied in this instance?

Reply:
RTD build failed: https://readthedocs.org/projects/sunpy/builds/12429511/

Reply:
(*Maybe it's the holidays I reckon...)

Reply:
Cool, all CI/CD checks passed now ❤️

Reply:
The trick @hayesla for the "subtitle" worked too, see: https://sunpy--4686.org.readthedocs.build/en/4686/generated/gallery/acquiring_data/goes_xrs_example.html#sphx-glr-generated-gallery-acquiring-data-goes-xrs-example-py